.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy to dev Cloudflare Worker
```

Comment:
I considered parametrizing the current build script, but thought it better to have separate ones

Reply:
that sounds better to me too 😅